lib/apiservers/portlayer/swagger.json

```diff
@@ -59,6 +59,115 @@
 				}
 			}
 		},
+		"/kv": {
```

Comment:
Given our prior conversation was expecting `"/kv/{key}"` and
```
{
	"name": "value",
	"in": "body",
	"schema": {
		"type": "string",
		"format": "binary"
	}
},
{
	"name": "key",
	"type": "string",
	"in": "path",
	"required": true
},
```

While our primary use for it currently is tags, restricting the API in that manner seems unnecessary.
Is there a reason for starting with a string restriction initially?